root.go

```diff
@@ -13,8 +13,8 @@ func rootCommand() *cobra.Command {
 		Use:     "subo",
 		Short:   "Suborbital Development Platform CLI",
 		Version: release.Version(),
-		Long: `Subo is the full toolchain for using and managing Suborbital Development Platform tools,
-including building WebAssembly Runnables and Atmo projects.`,
+		Long: `Subo is the full toolchain for using and managing Suborbital
+Development Platform tools.`,
```

Comment:
I don't think we need the linebreak here, and 'Suborbital Development Platform' can be replaced with 'Suborbital Extension Engine'